src/admin/components/CombineInput.vue

```diff
@@ -54,15 +64,15 @@
             value: {
                 type: Object,
                 default: {
-                    fixed: 0,
-                    percentage: 0
+                    fixed: '',
+                    percentage: ''
                 }
             },
         },
         data() {
             return {
-                fixed: this.formatPositiveValue( this.value.fixed ) ?? 0,
-                percentage: this.formatPositiveValue( this.value.percentage ) ?? 0
+                fixed: this.formatPositiveValue( this.value.fixed ) ?? '',
```

Comment:
this.value.fixed.trime() === '' ? this.formatPositiveValue( this.value.fixed ) : ''